cmd/fetch.go

```diff
@@ -10,11 +10,12 @@ import (
 )
 
 var fetchCmd = &cobra.Command{
-	Use:   "fetch (<NAME[:SEMVER|DIGEST]> | <TMID>)",
-	Short: "Fetches the TM by name or id",
-	Long:  `Fetches TM by name, optionally accepting semantic version or digest.`,
-	Args:  cobra.ExactArgs(1),
-	Run:   executeFetch,
+	Use:   "fetch <NAME>[:<SEMVER>] | <TMID>)",
+	Short: "Fetches a TM by name or id",
+	Long: `Fetches a TM by name, optionally accepting a semantic version.
```

Comment:
--help shows the long text. 
In the current long text it is not mentioned that you can also fetch by full ID.

Reply:
good point. fixed